corehq/apps/dump_reload/sql/filters.py

```diff
@@ -93,6 +93,55 @@ def querysets(self):
             yield queryset.filter(filter)
 
 
+class GetattrQueryset:
+    def __init__(self, queryset, attr):
+        self.queryset = queryset
+        self.attr = attr
+
+    def iterator(self):
+        for model in self.queryset:
+            print('-----', model)
+            yield getattr(model, self.attr)
+
+    def __iter__(self):
+        yield from self.iterator()
+
+    def __getattr__(self, item):
+        return getattr(self.queryset, item)
+
+    def __getitem__(self, item):
+        return self.queryset[item]
+
+
+class FilteredModelFieldIteratorBuilder(UnfilteredModelIteratorBuilder):
```

Comment:
nit: this name is a bit hard to grasp - maybe something like `RelatedModelIteratorBuilder`?  Alternatively, a short docstring would be useful in explaining what this is used for.